lib/proto/tupelo_rpc.proto

```diff
@@ -1,8 +1,51 @@
+// RUN `go generate` in this directory when updating the service.proto. Requires the protoc command to be in your path.
 syntax = "proto3";
 package walletrpc;
 
 option go_package = "walletrpc";
 
+message ProtoSetDataPayload {
+  string path = 1;
+  bytes value = 2;
+}
+
+message ProtoSetOwnershipPayload {
+  repeated string authentication = 1;
```

Comment:
is this what we call them in tupelo proper? Not addresses?

Reply:
yes, that's what they're called in the golang type declaration.